hydra-node/golden/RunOptions.json

```diff
@@ -8,33 +8,45 @@
-                "cardanoVerificationKeys": [
-                    "b/a/a.vk",
-                    "a/a/a.vk",
-                    "b/b/c.vk",
-                    "b/b.vk",
-                    "a/c/a.vk"
-                ],
-                "contestationPeriod": 11010,
-                "depositPeriod": 130,
+                "cardanoSigningKey": "a/a/a/b/b.sk",
+                "cardanoVerificationKeys": [],
+                "chainBackendOptions": {
+                    "contents": {
+                        "projectPath": "blockfrost-project.txt"
+                    },
+                    "tag": "Blockfrost"
+                },
+                "contestationPeriod": 43200,
+                "depositDeadline": 74,
                 "hydraScriptsTxId": [
-                    "88e381408dcd681c0f0bbd894c5202feede5c8790dbdff3a5de50b2fdeca0396",
-                    "52aea27bb01d5f1c3fdccde7d40f8d743cc397c0d1d810bfb1d96fd3ac934f09",
-                    "7ac013be0386ba006230e2f5ca7cba3cd6ffa141482090295316040a44f3baad",
-                    "9511f25390a2a1fb31ea3b0dc50a2cba229454bf91ac39a7d0f6c418531bd6cf"
+                    "0e7aa6a6b04903b07ee1c14b892dd41d79fcdb8d5150b379ee395e6fccd7ac94",
+                    "678a8fe9a4d9bae4beb9f04f3514a0ac4934a30213ad0e78adc386c8a3b40b6a",
+                    "4f87ef0eca6edfa9b58e9657b935ddce502c16cda64ce65076c458d130a28c06",
+                    "a6b6665b1d9c137c6660853971dc262f2a8eddf3c70e82c4558d08c1e9b4db42",
+                    "58085cd8f00178d12570caccff88b923545149b01201bee706e31ccd08b6b2fe",
+                    "f3f9c62b1702dc857bef7126f1c54ac2aea79610732b2840230abadfa37d3aac",
+                    "18b9f8f934bb1f85f37d12e1edadb8129deda7872fe472c7ede651ac4a72cc3a",
+                    "d0dbb06db756fee9a2e502cbd78f867bddea7dc1f0646fef43b0b5703ec3ed8d",
+                    "0016e2344ed38c643a727109e50d3461fa78d16931a3b5ab2b205b9039704b5c",
+                    "762283fe2ce87f27d30eff1f4dcb3b7fd3cd0e217e4b07f9f5db523cf2237a7f",
+                    "948cd38b0c12e73abfe7d9a650da29f7e6ae44b1b618c63f397786df046e413b",
+                    "ae16749c9326938539e93175b1195536bbf0f3c76ebb2b4811a35d82452384dc",
+                    "264bf4ac74f4e6db1bf61902ed1f0048f9bda859fec530fedee48fb3f9419bbc",
+                    "64b169ab192befd1223eadf5bcdd3f3729b4c45f4a5990ece3092a0e7bf052ec",
+                    "bca83bd1e872336dc73837a6c6a9eb40e71ef73dff765e4edc9207eef7eb28b2",
+                    "be269d5552683c9c1270083dfe3fc1131083dd4c99bae89762eb7151e722c232",
+                    "e8b34d73a29bf255f038c65c26ce1066d29e8d081d293ea4124a1ba841ddba9f",
+                    "18a22e4400bb478fec5dcad29b32aabd00b822aa721dccf8c9821354aabea153",
+                    "d0bafe586ebb838b9f8f8bb533892be0cb740a68d6f2b77b9f2d4f2c9ce08464",
+                    "b6fa856a9c21da5be600bda4da3b31dbbe213b0fe97f62340551e465fea3d963",
+                    "a726aedba74a0dc5f3ead4dc662fac9dfe1e566f88a9f7b9ebe7bf47e569df39"
```

Comment:
What's happening here? Why so many script tx ids?

Reply:
Nothing changed but I added `reasonablySized` generator so we don't end up with a huge file.